### PR TITLE
Getting it compiled with 2.0.81.1

### PR DESCRIPTION
**Pull request by FelixPetriconi**
These changes allow flint to be compiled with D compiler 2.0.81.1

### CODE REVIEW

**Comment by FelixPetriconi** [comment]
There are more changes on the way to get it compiled with the current compiler version.

**Comment by markisaa** [comment]
@FelixPetriconi - Thank you for contributing, but we are no longer merging any pull requests. This project is archived and no longer being supported, per the project's `README.md`.

**Comment by FelixPetriconi** [comment]
All changes done to get flint and flinttest compiled

File: Checks.d [hunk 1]
```diff
@@ -2406,7 +2406,7 @@ uint checkMutexHolderHasName(string fpath, Token[] v) {
     return 0;
   }
 
-  bool[string] mutexHolderNames = ["lock_guard":1];
+  bool[string] mutexHolderNames = ["lock_guard":1/*, "shared_lock":1, "unique_lock":1*/];
```
Comment (Orvid) [inline Checks.d]: Drop the extra commented members here.

**Pull request merged**